doctrine/associations.rst

```diff
@@ -146,7 +146,7 @@ the ``Product`` entity (and getter & setter methods):
 
             /**
              * @ORM\ManyToOne(targetEntity="App\Entity\Category", inversedBy="products")
-             * @ORM\JoinColumn(nullable=false)
+             * @ORM\JoinColumn(nullable=true)
```

Comment:
In that case, can be removed the annotation (it's already the default value)

Reply:
@yceruto you meant remove the entire `@ORM\JoinColumn` instead of leaving it empty `@ORM\JoinColumn()`, right?

Reply:
Right, completely.